share/completions/zpool.fish

```diff
@@ -167,11 +166,7 @@ function __fish_zpool_list_rw_properties
     echo -e "listsnaps\t"(_ "Display snapshots even if 'zfs list' does not use '-t'")" (on, off)"
     echo -e "version\t"(_ "On-disk version of pool")" (VERSION)"
     # Feature properties
-    for featureLong in (zpool list -o all | tr -s "[:blank:]" | tr ' ' '\n' | tr "[:upper:]" "[:lower:]" | grep '^feature@')
-        set -l featureShort (echo $featureLong | sed -e 's/feature@\(.*\)/\1/g')
-        set featureLong (echo -e "$featureLong\t")
-        printf (_ "%sEnable the %s feature\n") $featureLong $featureShort
-    end
+    zpool list -o all | string replace -ra ' +' '\n' | string lower | string replace -rf '^feature@(.*)' '$1\tEnable this feature'
```

Comment:
This works fine for me. `man zpool-features` contains a semi-brief description of each feature, I don't know if that can be used to provide more useful information here. I'm not a fan of using a description of the *command* as a description of the *completion* where it gets repeated for each completion (as discussed elsewhere), either:

```
mqudsi@freebsd /u/h/m/fish-shell> zpool list -o all | string replace -ra ' +' '\n' | string lower | string replace -rf '^feature@(.*)' '$1\tEnable this feature'
async_destroy   Enable this feature
empty_bpobj     Enable this feature
lz4_compress    Enable this feature
multi_vdev_crash_dump   Enable this feature
spacemap_histogram      Enable this feature
enabled_txg     Enable this feature
hole_birth      Enable this feature
extensible_dataset      Enable this feature
embedded_data   Enable this feature
bookmarks       Enable this feature
filesystem_limits       Enable this feature
large_blocks    Enable this feature
large_dnode     Enable this feature
sha512  Enable this feature
skein   Enable this feature
device_removal  Enable this feature
obsolete_counts Enable this feature
zpool_checkpoint        Enable this feature
spacemap_v2     Enable this feature
```

Reply:
Yeah, I agree.